### PR TITLE
Sdk nightly download mapper

### PR DESCRIPTION
**Pull request by abaumgarten**
## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/30826

## Description
Added the testing of upload and download to the sdk nightly

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1165,7 +1167,6 @@ jobs:
           name: Run lint
           command: demisto-sdk lint -i Packs/HelloWorld/
       - *upload_entities_to_cortex_xsoar
-      - *destroy_instances
```
Comment (guyfreund) [inline .circleci/config.yml:1168]: Why deleted?

File: .circleci/config.yml [hunk 2]
```diff
@@ -1155,6 +1156,7 @@ jobs:
             demisto-sdk download -o Packs/HelloWorld/ -i HelloWorld-Test --insecure -f
             demisto-sdk download -o Packs/HelloWorld/ -i "Hello World Incident Test" --insecure -f
             demisto-sdk download -o Packs/HelloWorld/ -i "Hello World Alert Test" --insecure -f
+            demisto-sdk download -o Packs/HelloWorld/ -i "Hello World Test - Incoming Mapper" --insecure -f
```
Comment (guyfreund) [inline .circleci/config.yml:1159]: update the name of the step in line `1149`

File: Tests/sdknightly/create_entities_for_nightly_sdk.py [hunk 3]
```diff
@@ -102,6 +102,36 @@ def create_incident_type(path: Path, layout_name: str) -> str:
...
+        os.mkdir(dest_mapper_path)
+
+    mapper_path = dest_mapper_path / 'classifier-mapper-incoming-HelloWorldTest.json'
+    with open(mapper_path, 'w+') as stream:
+        json.dump(mapper, stream, indent=4)
+    return str(mapper_path)
+
+
 def main():
     parser = argparse.ArgumentParser(description="Creates incident field, incident type and a layout in a given pack.")
```
Comment (guyfreund) [inline Tests/sdknightly/create_entities_for_nightly_sdk.py:136]: update description

**Review by guyfreund — CHANGES_REQUESTED** [review]
- Why do we have changes in `package.json` and `package-lock.json`?
- Please attach a successful sdk nightly run

File: Tests/sdknightly/create_entities_for_nightly_sdk.py (hunk 3 above)
Comment (abaumgarten) [reply Tests/sdknightly/create_entities_for_nightly_sdk.py]: Done

File: .circleci/config.yml (hunk 1 above)
Comment (abaumgarten) [reply .circleci/config.yml]: by mistake, adding it.

File: .circleci/config.yml (hunk 2 above)
Comment (abaumgarten) [reply .circleci/config.yml:1159]: done

File: package.json [hunk 4]
```diff
@@ -20,7 +20,7 @@
   },
   "homepage": "https://github.com/demisto/content#readme",
   "dependencies": {
-    "@mdx-js/mdx": "^1.6.19",
+    "@mdx-js/mdx": "^1.6.21",
```
Comment (jochman) [inline package.json:23]: why is that?

**Review by jochman — CHANGES_REQUESTED** [review]
why you updated the mdx package?

File: package.json (hunk 4 above)
Comment (abaumgarten) [reply package.json:23]: @guyfreund

Comment (abaumgarten) [reply package.json]: Done

**Comment by abaumgarten** [comment]
A successful nightly run : 
https://app.circleci.com/pipelines/github/demisto/content/46683/workflows/c21751cf-7e79-42f7-aa35-46587688094d/jobs/200164

**Review by guyfreund — APPROVED** [review]
Wait for @jochman's approval too

**Review by jochman — APPROVED** [review]

**Pull request merged**